samples/Calculator/tests/Calculator.Server.IntegrationTests/IntegrationTests.cs

```diff
@@ -0,0 +1,38 @@
+namespace Calculator.Server.IntegrationTests;
+
+[TestFixture]
+public class IntegrationTests
+{
+    private readonly HttpClient _client;
+
+    public IntegrationTests()
+    {
+        _client = new HttpClient()
+        {
+            BaseAddress = new Uri(" http://localhost:5291"),
+        };
+    }
+
+    [Test]
+    [TestCase("/add/3/5", "8")]
+    [TestCase("/multiply/3/5", "15")]
+    [TestCase("/substract/3/5", "-2")]
+    [TestCase("/devide/35/5", "7")]
+    [Parallelizable(ParallelScope.All)]
+    public async Task TestOperations(string input, string output)
+    {
+        try 
+        {
+            // Act
+            var response = await _client.GetAsync(input);
+
+            // Assert
+            response.EnsureSuccessStatusCode(); // Status Code 200-299
+            Assert.That(await response.Content.ReadAsStringAsync(), Is.EqualTo(output));
+        }
+        catch (HttpRequestException e)
+        {
+            Assert.Ignore(e.Message);
```

Comment:
why ignore?

Reply:
When Server is not running I want to Skip test - not fail. It is important in scenario when you do "dotnet test" on solution level.

Reply:
ok I though that the test start custom server, fine so

Reply:
If test would start custom server then we can't show how to collect in e2e scenario. I want server to be started separately under dotnet-coverage